sirepo/package_data/template/elegant/examples/bunchCompLSC.json

```diff
@@ -3932,7 +3932,7 @@
         "rpnVariables": [],
         "simulation": {
             "activeBeamlineId": 28,
-            "folder": "/",
+            "folder": "/Examples",
```

Comment:
I'd recommend leaving the example files unchanged - the fixup will automatically correct them. That makes less change in the pull request as well.